src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/TryHandler.java

```diff
@@ -42,10 +43,31 @@ public TryHandler(IndentationCheck indentCheck,
 
     @Override
     public IndentLevel getSuggestedChildIndent(AbstractExpressionHandler child) {
+
         if (child instanceof CatchHandler
             || child instanceof FinallyHandler) {
             return getIndent();
         }
+
+        final DetailAST resSpecAst = getMainAst().getFirstChild();
+        if (resSpecAst.getType() == TokenTypes.RESOURCE_SPECIFICATION) {
+            final DetailAST resourcesAst = resSpecAst.findFirstToken(TokenTypes.RESOURCES);
+            final IndentLevel expIndent = new IndentLevel(getIndent(), getBasicOffset());
+            DetailAST resourcesAstChild = resourcesAst.findFirstToken(TokenTypes.RESOURCE);
+
+            for (int i = 0; i < resourcesAst.getChildCount(); i++) {
+
+                if (resourcesAstChild.getType() == TokenTypes.RESOURCE
+                    && resourcesAstChild.getLineNo() != resSpecAst.getLineNo()
+                    && resourcesAstChild.getColumnNo()
+                        != Integer.parseInt(expIndent.toString())) {
+                    logError(resourcesAstChild, "",
```

Comment:
This probably relates directly to my comment about the no warning in the tests.
You are just making sure where the resource starts on the line, and not all the lines of the resource.
If we call the line wrapping handler here for the entire 1 resource, it would handle the wrapping for us.
What do you think of this?